src/model_import.jl

```diff
@@ -0,0 +1,40 @@
+using YAML
+using Requests
```

Comment:
Similar to above, let's move this to src/Dolo.jl and have it read `using Requests: get`